REFERENCE.md

```diff
@@ -7,6 +7,7 @@
 ### Functions
 
 * [`extlib::cache_data`](#extlibcache_data): Retrieves data from a cache file, or creates it with supplied data if the file doesn't exist
+* [`extlib::cidr_to_network`](#extlibcidr_to_network): Converts an CIDR address of the form 192.168.0.1/24 into its network address (also known as net address)
```

Comment:
`an CIDR` → `a CIDR`

How about using modern IP instead of legacy for the example? `2001:DB8::/32` (`192.0.2.0/24` could be used for legacy if desired, reference: <https://en.wikipedia.org/wiki/Reserved_IP_addresses>)

Reply:
thanks, fixed it